File: auto-docs/executables/r/Bluered_heatmap.r
library(plotly)

p <- plotly(username='TestBot', key='r1neazxo9w')

size <- 50
z <- matrix(0, size, size)
for(r in 1:size) {
    for(c in 1:size) {
        z[r, c] <- r+c
    }
}

data <- list(
  list(
    z = z, 
    scl = "Bluered", 
    type = "heatmap"
  )
)
layout <- list(title = "Bluered")

response <- p$plotly(data, kwargs=list(layout=layout, filename="Bluered-heatmap", fileopt="overwrite", auto_open="FALSE"))
url <- response$url
filename <- response$filename

File: auto-docs/executables/matlab/Bluered_heatmap.m
signin('TestBot', 'r1neazxo9w')

size = 50;
z = zeros(size, size);
for r = 1:size
    for c = 1:size
        z(r,c) = sqrt(r*c/size^2)
    end
end
data = {...
  struct(...
    'z', z, ...
    'scl', 'Bluered', ...
    'type', 'heatmap')...
};
layout = struct('title', 'Bluered');

response = plotly(data, struct('layout', layout, 'filename', 'Bluered-heatmap', 'fileopt', 'overwrite', 'auto_open', 'false'));
plot_url = response.url